elsie/boxitem.py

```diff
@@ -2,6 +2,13 @@
 
 
 class BoxItem(BoxMixin):
+    """
+    Leaf child of the box layout hierarchy.
+
+    When you call a method from `BoxMixin` that creates a new box (like text, image, etc.), it will
```

Comment:
box => box/boxitems (because text(), image() does not create boxes but boxitems)